docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -331,6 +359,100 @@ The example below defines the following table access policy:
       ]
     }
 
+.. _system-file-function-rules:
+
+Function rules
+^^^^^^^^^^^^^^
+
+These rules control the user's ability to execute SQL functions,
+such as :doc:`aggregate functions </functions/aggregate>`, scalar functions,
+:doc:`table functions </functions/table>` and :doc:`window functions </functions/window>`.
+
```

Comment:
Somewhere here we should add that by default all functions are allowed.

Reply:
```
These rules control the user's ability to execute SQL all function kinds, such as...
```
?

Reply:
Sure .. but also say .. by default all functions are available.

Reply:
When user does not define any `functions` rule then all functions are available. That's how the rest of rules works as well.